operations/iscsi_sbps/Managing_selective_node_personalization.md

```diff
@@ -0,0 +1,131 @@
+# Managing Selective Node Personalization
+
+## Overview
+
+The selective iSCSI worker node personalization feature allows administrators to specify which worker NCNs are enabled as
+iSCSI targets. The mechanism by which an administrator specifies this is an HSM group named `iscsi_worker`.
+The existence of this group on the system means that selective iSCSI worker node personalization is enabled.
+If the group does not exist, then the feature is disabled, and all worker nodes will be enabled as iSCSI targets.
+While it is technically possible to create this group and leave it empty, this will mean that no worker nodes will be
+enabled as iSCSI targets, which in turn will mean that no managed nodes will be able to boot. Therefore CSM generally
+treats it as an error if the HSM group exists but it contains no worker nodes.
+
+The [Group commands](#group-commands) section goes over the commands used to create, remove, or modify the group.
+The [Procedures](#procedures) section discusses the different contexts in which this may be done, and any additional
+steps that must be performed to effectuate the changes.
+
+## Group commands
+
+Example commands to manage selective worker node personalization for iSCSI SBPS:
+
+(`ncn-mw#`) Create HSM group:
+
+```bash
+cray hsm groups create --label iscsi_worker --description "iscsi node personalization" --members-ids x3000c0s5b0n0
+```
+
+Example output:
+
+```json
+[[results]]
+URI = "/hsm/v2/groups/iscsi_worker"
+```
+
+HSM group `iscsi_worker` created with xname `x3000c0s5b0n0` added.
+
+(`ncn-mw#`) Adding one more xname of worker node is as below:
+
+```bash
+cray hsm groups members create --id x3000c0s18b0n0 iscsi_worker
+```
+
+Example output:
+
+```json
+[[results]]
+URI = "/hsm/v2/groups/iscsi_worker/members/x3000c0s18b0n0"
+```
+
+(`ncn-mw#`) The group members of `iscsi_worker` can be listed as below:
+
+```bash
+cray hsm groups members list iscsi_worker
+```
+
+Example output:
+
+```json
+ids = [ "x3000c0s5b0n0", "x3000c0s18b0n0",]
+```
+
+(`ncn-mw#`) Deleting the worker node from `iscsi_worker` group:
+
+```bash
+cray hsm groups members delete x3000c0s18b0n0 iscsi_worker
+```
+
+Example output:
+
+```json
+code = 0
+message = "deleted 1 entry"
+```
+
+(`ncn-mw#`) Checking the group after node deletion which shows empty list as below:
+
+```bash
+cray hsm groups members list iscsi_worker
+ids = []
+```
```

Comment:
Since an empty group is not a supported configuration, I think we don't need to include this example.

```suggestion
```